css/tutorial.styl

```diff
@@ -25,10 +25,14 @@
     > .media-card-content
       @extends .content-container
 
+    img
+      max-width: 100%
```

Comment:
Ok, so this is targeting images in the markdown instead of uploaded image media linked to the tutorial step (the `media-card-media`). The tutorial for Amazon Aerobotany is using images its markdown content, which you're right, should have a max-width applied too. The `img` styling here should be indented to be a child of `media-card-content` so that it's clear that the style is to target images in the markdown.